### ISSUE TITLE
MSU fails to install with space in path

### ISSUE THREAD

**Issue by frankhalligan** [chocolatey/choco#1177]
### What You Are Seeing?
MSU-based package fails to install if there is a space in the Windows username

### What is Expected?
Install to succeed

### How Did You Get This To Happen? (Steps to Reproduce)
Windows 7 Professional 6.1.7601
Windows Username: Frank Halligan
Chocolatey: 0.10.3
~~~sh
cinst powershell
~~~
Also occurs with "cinst rsat" (and presumably any other msu-based packages)

### Output Log
Full debug log: https://gist.github.com/frankhalligan/096ab580c40307a1dfe71e6c1b2151d2

Specific problem:
~~~sh
Setting RunAs for elevation
Command ["C:\Windows\System32\wusa.exe" C:\Users\frank halligan\AppData\Local\Temp\chocolatey\PowerShell\5.1.14409.20170301\PowerShellInstall.MSU /quiet /norestart /log:"C:\Users\frank halligan\AppData\Local\Temp\chocolatey\PowerShell.Install.evtx" ] exited with '87'.
ERROR: Running ["C:\Windows\System32\wusa.exe" C:\Users\frank halligan\AppData\Local\Temp\chocolatey\PowerShell\5.1.14409.20170301\PowerShellInstall.MSU /quiet /norestart /log:"C:\Users\frank halligan\AppData\Local\Temp\chocolatey\PowerShell.Install.evtx" ] was not successful. Exit code was '87'. See log for possible error messages.
~~~
Note the lack of quotation marks around
~~~sh
C:\Users\frank halligan\AppData\Local\Temp\chocolatey\PowerShell\5.1.14409.20170301\PowerShellInstall.MSU
~~~

**Comment by DarwinJS** [comment]
I asked Frank to put file this issue.

The line of code for MSU appears to be here: https://cloudywindows.com/post/oneliners-for-procmon-debugging-of-aws-cloud-formation-automation/

But EXE might be affected here: https://cloudywindows.com/post/oneliners-for-procmon-debugging-of-aws-cloud-formation-automation

MSIs look to be handled correctly here: https://github.com/chocolatey/choco/blob/c798ae3a1b79101814599143a0ed416a69dc13ff/src/chocolatey.resources/helpers/functions/Install-ChocolateyInstallPackage.ps1#L237

**Comment by ferventcoder** [comment]
@frankhalligan @DarwinJS Thanks for logging that. This is now fixed and will be available in 0.10.4.

**Comment by ferventcoder** [comment]
Just looked and there is no additional followup for licensed customers - they are already good here.
